RELEASE_NOTES.md

```diff
@@ -1,5 +1,189 @@
+  the "experimental" feature flag during compilation.
+* The "biome" and "postgres" features are now available with the "stable"
+  feature flag (instead of "experimental") and will be included in the default
+  and stable Docker images published at
+  [splintercommunity](https://hub.docker.com/u/splintercommunity).
+* There is a new [splinter-ui](https://github.com/Cargill/splinter-ui)
+  repository for Canopy and saplings.
+
+### Deprecations and Breaking Changes
+
+* Biome routes for user keys no longer require a user ID.
+    - The endpoint `biome/users/{user_id}/keys` is now `biome/keys`
+    - The endpoint `biome/user/{user_id}/keys/{public_key}` is now
+      `biome/keys/{public_key}`
+* The `splinter circuit create` subcommand is now `splinter circuit propose`.
+* The `splinter node alias` subcommands have been removed. This functionality
+  will be replaced by `splinter circuit template` subcommands in an upcoming
+  release.
+
+For upgrade information, see [Upgrading to Splinter 0.3.15 from Splinter 0.3.14](https://github.com/Cargill/splinter-docs/blob/master/docs/upgrading/splinter-v0.3.15-from-v0.3.14.md).
+
+### libsplinter
+
+* Move the "biome" and "postgres" features from experimental to stable. They
+  can be enabled with the “stable” feature flag (instead of the “experimental”
+  feature flag), in addition to being enabled individually. These features
+  will be used by the default and stable Docker images published at
+  [splintercommunity](https://hub.docker.com/u/splintercommunity).
+* Remove the following features (no longer available as compilation options),
+  because the functionality is now available by default:
+    - `biome-rest-api`
+    - `circuit-read`
+    - `proposal-read`
+* Remove the following features (no longer available as compilation options):
+    - `database` - Redundant; its functionality can be accessed with the
+       "postgres" feature,
+    - `json-web-tokens` - Deemed unnecessary, because using authorization tokens
+       throughout the Biome REST API is always required,
+* Clean up and expand the transport module:
+    - Remove unused errors, PollError and StatusError, from the transport
+      module.
+    - Remove transport status enum.
+    - Implement `Display` for transport errors.
+* Refactor the database module:
+    - Refactor the constructor for `ConnectionPool` to `ConnectionPool::new_pg`
+      to support multiple backends.
+    - Replace `database::DatabaseError` with `database::ConnectionError`
+      because the only valid database errors were connection related.
+
+#### Testing
+
+* The REST API tests now shut down after the tests finish.
+
+#### Biome
+
+* Change two Biome routes so that the user ID is not required in the route
+    - `biome/users/{USER-ID}/keys` has changed to to `biome/keys`
+    - `biome/users/{USER-ID}/keys/{public_key}` has changed to
+      `biome/keys/{public_key}`
+   The user ID is now derived from the provided access token.
```

Comment:
This is supposed to be on its own line, but it appears that github markdown won't do that. As a work-around, I propose moving this line up with the first one: 

> * Change two Biome routes so that the user ID is not required in the route. The user ID is now derived from the provided access token.